recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "3.0.0" %}
+{% set version = "3.0.1" %}
 
 package:
   name: compliance-checker
   version: {{ version }}
 
 source:
   fn: compliance-checker-{{ version }}.tar.gz
-  url: https://github.com/ioos/compliance-checker/archive/{{ version }}.tar.gz
-  sha256: f49d58d44ba7e75509317dd2b8ec5ee68e5d0e4a90ccab66ae28895e5a7c3276
+  url: https://codeload.github.com/ioos/compliance-checker/tar.gz/{{version}}
```

Comment:
Why the URL change? `https://github.com/ioos/compliance-checker/archive/{{ version }}.tar.gz` should work.